probe_scraper/transform_probes.py

```diff
@@ -275,48 +275,74 @@ def metrics_equal(def1, def2):
      ))
 
 
-def update_or_add_metric(repo_metrics, commit_hash, metric, definition, commit_timestamps):
-    # If we've seen this metric before, check previous definitions
-    if metric in repo_metrics:
-        prev_defns = repo_metrics[metric][HISTORY_KEY]
+def ping_equal(def1, def2):
+    return all((
+        def1.get(l) == def2.get(l)
+        for l in {
+            'bugs',
+            'data_reviews',
+            'description',
+            'notification_emails',
+            'include_client_id',
+            'send_if_empty',
+        }
+    ))
```

Comment:
We should make this compare all fields. In the past we've had issues where a new field is added and then a change in it isn't present in the probe-info-service.